config/lava/iec-security/iec-security.jinja2

```diff
@@ -0,0 +1,13 @@
+- test:
+    name: {{ plan }}
+    description: "cyber security standard tests such as IEC-62443-4-2"
+    timeout:
+      minutes: 20
+    definitions:
+    - repository: https://gitlab.com/cip-project/cip-testing/cip-security-tests.git
```

Comment:
It would be useful to set up a fork in the KernelCI GitHub project to be able to atomically update the revision during the weekly production update, to avoid having the test definitions changing while a kernel revision is being tested.

Reply:
yes, i see.

Reply:
made the fork